src/lib/env.js

```diff
@@ -9,13 +9,6 @@ const yargs = _yargs(hideBin(process.argv));
  * @extends Object
  * @param {number} logLevel
  * @param {boolean} logColor
- * @param {string} psqlHost
- * @param {number} psqlPort
- * @param {string} psqlUser
- * @param {string} psqlPassword
- * @param {string} psqlDatabase
- * @param {number} psqlPoolMin
- * @param {number} psqlPoolMax
  * @param {string} apiURL
  * @param {string} webhookKey
```

Comment:
Are we using the webhook still? I dont believe so. This an its usage needs to be reviewed and removed if needed

Reply:
webhook key is being called by fetch.js, which in turn is being called by index.js